### DISCUSSION TITLE
Initial Setup Guide

### DISCUSSION THREAD

**Discussion by wreed87** [davidmallasen/LiveChess2FEN#6] [Ideas]
Hi,

I am interested in trying this framework using my Jetson Nano. However, I lack the necessary knowledge to understand how can this be set up even after reading the instructions in Readme.

Is there a chance that a video/comprehensive guide could be made to explain the steps required to install the framework?

Thanks,

N.B. great work!

**Comment by davidmallasen** [comment]
Hello @wreed87

I don't have the time to do this in the near future. However, it should be well documented in the nvidia forums. For example here: https://docs.nvidia.com/deeplearning/frameworks/install-tf-jetson-platform/index.html

If you manage to install everything and write down the steps you take, I can add them to the repo for other people to benefit as well.

Thanks for your interest!

**Reply by jameel-shorosh-silk** [reply]
is there already an installation instruction? and maybe some examples how to use this framework?

**Reply by davidmallasen** [reply]
You can find the setup and how to get things running in the README of the project. Let me know if there is something not clear. If you see some step missing, feel free to open a pull request as well fixing it.

**Reply by jameel-shorosh-silk** [reply]
can I install this project on my laptop running ubuntu and use it to convert real-time chess boards pictures into FEN? or I need a special hardware for this? which section to follow in the README.md? "Jetson Nano 2GB install" or "Setup" ?

**Reply by davidmallasen** [reply]
Yes. You can use a normal laptop running ubuntu. In that case follow the https://github.com/davidmallasen/LiveChess2FEN#setup but with `requirements_pc.txt` instead of `requirements.txt`

**Reply by sailoukik** [reply]
can i use this on raspberry pi 5 with the raspbian os ? using the requirements pc ?

**Reply by davidmallasen** [reply]
Most likely yes! You'll have to use ONNX to run the CNNs instead of TensorRT. Let me know how it goes once you get it running and I can add it to the README.
